### PR TITLE
fix: loading modules

### PR DESCRIPTION
**Pull request by lingy1028**
Raise the PR to fix issue #25

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -10,16 +10,13 @@ snmp:
       - "my-area"
   mibs:
     url:
-      - "file:///usr/share/snmp/mibs"
+      # - "file:///usr/share/snmp/mibs"
```
Comment (xynazog) [inline config.yaml:13]: Can you uncomment this?

File: splunk_connect_for_snmp_traps/manager/translator.py [hunk 2]
```diff
@@ -21,8 +20,12 @@ def build_mib_compiler(self, load_list):
         mibViewController = view.MibViewController(mibBuilder)
         compiler.addMibCompiler(mibBuilder, sources=snmp_config["mibs"]["url"])
 
-        for module in self._load_list:
-            mibBuilder.loadModules(module)
+        try:
```
Comment (xynazog) [inline splunk_connect_for_snmp_traps/manager/translator.py:23]: Thanks for adding try-catch 😀

**Review by xynazog — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (lingy1028) [reply config.yaml]: sure.

**Review by xynazog — APPROVED** [review]

**Pull request merged**